src/components/FormSelect/FormSelect.js

```diff
@@ -35,10 +35,10 @@ const FormSelect = ({
         <option value="" disabled>
           {placeholder}
         </option>
-        {options.map((value) => {
+        {options.map((option) => {
```

Comment:
destructuring is recommended
```suggestion
        {options.map(({ key, value, label }) => {
```